.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
You could add the `--format documentation` flag to make your tests more readable.